Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -148,6 +151,33 @@ public static ProcessService getInstance() {
         return convertJSONObjectsToDTOs(findAllDocuments(sort, offset, size), false);
     }
 
+    @Override
+    public List<ProcessDTO> findAll(String sort, Integer offset, Integer size, Map filters) throws DataException {
+
```

Comment:
It looks this method is not called in ProzessverwaltungForm - so it displays all processes and templates.